finaps/libs/frontend/routing/src/lib/frontend-routing.module.ts

```diff
@@ -0,0 +1,31 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router'; // CLI imports router
+import { HomeComponent } from '../../../feat/home/src/lib/home/home.component';
+
+const routes: Routes = [
+  {
+    path: '',
+    component: HomeComponent,
+  },
+  {
+    path: 'strava',
+    loadChildren: () =>
+      import('../../../feat/strava/src/lib/frontend-feat-strava.module').then(
+        (m) => m.FrontendFeatStravaModule
```

Comment:
Not sure how I feel about the naming convention but it's fine for now but lets discuss later ;) But my point is that the feature will be imported from `@finaps/frontend/feat/strava` so not sure if we need to be this verbose on the names of things like the module

Reply:
Also the import shouldnt be relative for `HomeComponent` :D

Reply:
Lastly I'm not sure if I like that we have `@finaps` at the beginning of every import but again its for a discussion rather than anything so just fix the relative import here hahahah